python/ql/test/experimental/dataflow/model-summaries/NormalDataflowTest.ext.yml

```diff
@@ -18,4 +18,6 @@ extensions:
       - ["foo", "Member[MS_spread]", "Argument[0]", "ReturnValue.TupleElement[0]", "value"]
       - ["foo", "Member[MS_spread]", "Argument[1]", "ReturnValue.TupleElement[1]", "value"]
       - ["foo", "Member[MS_spread_all]", "Argument[0]", "ReturnValue.TupleElement[0,1]", "value"]
+      - ["foo", "Member[MS_Class].Call", "Argument[0, x:]", "ReturnValue", "value"]
```

Comment:
I suggest you rewrite this to store the Argument 0/x: into an attribute of the return value, that's more representative of what would usually happen in a model (I assume).

Reply:
Done. Also updated to latest main. Sorry for the force push; fortunately the changes are quite contained.